.github/actions/rng/action.yml

```diff
@@ -0,0 +1,25 @@
+name: "RNG"
+description: "Generate random bytes using /dev/urandom. WARNING: only use for non-cryptographic purposes (the results will show in logs)."
+inputs:
+  length:
+    description: "Length to generate (in bytes)."
+    required: true
+outputs:
+  random:
+    description: "The output of the RNG (hex-encoded)."
+    value: "${{ steps.prng.outputs.result }}"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Generate random
+      id: prng
+      env:
+        LENGTH: "${{ inputs.length }}"
+      run: |
+        set -euo pipefail
+
+        value=$(head -c"$LENGTH" /dev/urandom | xxd -p)
```

Comment:
nit: I think you can just do
```
xxd -l "$LENGTH" -p /dev/urandom
```

Also, both the above command and one you have here produces something longer than `$LENGTH` characters. Maybe something like this to ensure it's LENGTH characters long?
```
xxd -p -l $LENGTH /dev/urandom | head -c$LENGTH
```

Reply:
ouch, `length` was intended to be the length of the raw random value (since this is directly related to the entropy), not the encoded one. I think I should rename the input argument then, or have a better description for it.
What would you suggest?

Reply:
Thanks. I'll use your first command. `length` is supposed to be the number of bytes of raw data not the length of the encoded data. I'll update the description to clarify

Reply:
I also found that we need to pass `-c $LENGTH` to avoid CR during display.